src/Configurator/ContainerConfigurator.php

```diff
@@ -76,13 +74,45 @@ private function addParameters(array $parameters)
             $parametersLines[] = "parameters:\n";
         }
         foreach ($parameters as $key => $value) {
-            // FIXME: var_export() only works for basics types, but we don't have access to the Symfony YAML component here
-            $parametersLines[] = sprintf('    %s: %s%s', $key, var_export($value, true), "\n");
+            if (\is_array($value)) {
+                $parametersLines[] = sprintf("    %s:\n%s", $key, $this->dumpYaml(2, $value));
+                continue;
+            }
+            $parametersLines[] = sprintf("    %s: '%s'%s", $key, addslashes($value), "\n");
```

Comment:
What about using str_replace instead as we only need to escape quotes?

Reply:
Yes, I think it will be better because i juste thinking it's not a slash for escape in yaml